e2e/wdio/platform/tests/checkbox.e2e-spec.ts

```diff
@@ -53,59 +53,60 @@ describe('Checkbox test suite', () => {
 
     describe('check binary checkbox used with form examples', () => {
         // TODO: Unskip after fix
-        xit('should check binary checkbox in template driven form', () => {
+        it('should check binary checkbox in template driven form', () => {
 
-            waitForElDisplayed(binaryTempCheckbox, 0);
+            //waitForElDisplayed(binaryTempCheckbox, 0);
             // check checkbox labels
             for (let i = 0; 3 > i; i++) {
-                checkIfDisabled(binaryTempCheckbox, 'ng-reflect-is-disabled', 'false', i);
+                //scrollIntoView(binaryTempCheckbox, i);
```

Comment:
please recheck if you need to keep this commented code, and if waitForDisplay is needed or not as well.